### PR TITLE
refactor: clean up code from previous transaction scheme

### PR DESCRIPTION
**Pull request by ckniffen**
## High Level Overview of Change

Removes the code that previously powered the old transaction scheme now that all transaction types have been refactored.

### Type of Change

- [x] Refactor (non-breaking change that only restructures code)

### CODE REVIEW

File: src/containers/shared/components/TxDetails.tsx [hunk 1]
```diff
@@ -1,148 +1,17 @@
-import React, { ReactElement } from 'react'
-import PropTypes from 'prop-types'
-import { useTranslation, withTranslation } from 'react-i18next'
-import { Amount } from './Amount'
+import React from 'react'
 import { transactionTypes } from './Transaction'
 
-interface Instructions {
```
Comment (ckniffen) [inline src/containers/shared/components/TxDetails.tsx:7]: Each tx defines what will be on the instructions.  This is redundant.

File: src/rippled/lib/txSummary/index.js [hunk 2]
```diff
@@ -1,16 +1,8 @@
...
 const getInstructions = (tx, meta) => {
   const type = tx.TransactionType
-  // Locate the transaction parser which returns "instructions" to be used in transaction specific components
-  // TODO: Remove summarize[type] lookup once all transactions have been moved to the new definition style
-  const mappingFn = transactionTypes[type]?.parser
-    ? transactionTypes[type]?.parser
-    : summarize[type]
-
-  return mappingFn ? mappingFn(tx, meta) : {}
+  return transactionTypes[type]?.parser(tx, meta) || {}
```
Comment (mvadari) [inline src/rippled/lib/txSummary/index.js:5]: Thoughts on something like this for a better default case if there's a parser error?
```suggestion
  try {
    return transactionTypes[type]?.parser(tx, meta)
  catch {
    return tx
  }
```

Or should that be handled in a separate PR? I guess the parsers would all need to change as well.

**Review by mvadari — APPROVED** [review]

**Review by pdp2121 — APPROVED** [review]

File: src/rippled/lib/txSummary/index.js (hunk 2 above)
Comment (ckniffen) [reply src/rippled/lib/txSummary/index.js:5]: Going with a separate PR.  I will create a ticket for this.
